aries-site/src/pages/index.js

```diff
@@ -1,69 +1,116 @@
 const pageDetails = getPageDetails(title);
-const cards = getCards();
-const featured = [
-  'Designer Guidance',
-  'Forms',
-  'Header',
-  'Human Centered',
-  'Developer Guidance',
-  'Button',
-  'Navigation',
-  'Icons',
-  'Search',
-  'Typography',
-];
 
-const featuredCards = [];
-const cardOrder = [];
+const calcPad = size => {
+  const val = size !== 'small' ? 'xlarge' : 'large';
+  return val;
+};
+
+// These make a box width limited to xxlarge but centered
+const widthProps = { width: { max: 'xxlarge' }, margin: 'auto' };
 
-cards.forEach(card => {
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+const Intro = ({ children }) => {
+  const size = useContext(ResponsiveContext);
+  return size === 'small' ? (
+    <Box>
+      <Hero height={{ max: '292px' }} margin={{ bottom: '-24px' }} />
+      <Card background="none" elevation="none">
+        {children}
+      </Card>
+    </Box>
+  ) : (
+    <Stack guidingChild="last">
+      <Box
+        align="start"
+        justify="between"
+        pad={{ horizontal: calcPad(size) }}
+        direction="row"
+        fill
+        {...widthProps}
+      >
+        <Box width="small" />
+        <Hero />
+      </Box>
+      <Box height={{ min: 'medium' }} justify="center" {...widthProps}>
+        <Grid
+          gap="large"
+          columns={size === 'small' ? ['auto'] : ['3/4', 'auto']}
+        >
+          <Card background="none" elevation="none">
+            {children}
+          </Card>
+          {size !== 'small' && (
+            <Card background="none" elevation="none" height="small" />
+          )}
+        </Grid>
+      </Box>
+    </Stack>
+  );
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-cardOrder.splice(0, 0, ...featuredCards);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
-        </Box>
-      </PageIntro>
-      <CardGrid cards={cardOrder} />
-    </Box>
-  </Layout>
-);
+  return (
+    <Layout title={title} isLanding pad={{}} width={{}}>
```

Comment:
```suggestion
    <Layout title={title} isLanding pad="none" width="100%">
```
effectively the same effect (would need to confirm locally), but feels a little strange to pass an empty object.

Reply:
soft opinion here, shouldn't change visual behavior regardless

Reply:
seeing your PropTypes for width and pad in Layout.js now, fine leaving as is with object in that case.